### ISSUE TITLE
Add `alias` and `unalias` into scripting

### ISSUE THREAD

**Issue by bartekleon** [bitburner-official/bitburner-src#865]
It would be a nice QoL to have `alias` and `unalias` functions available in scripting.

**Comment by Snarling** [comment]
Scripts already cannot input other terminal commands (without document manipulation), so they have no way to interact with the newly formed aliases. Can you give an example of the desired use case?

**Comment by bartekleon** [comment]
TLDR: I'm writing lots of scripts I want to alias and making it programatically (all scripts from some folder) would be nice

So my problem is a result that writing `run ping.js` (even with autocomplete) is just annoying. Yes I can alias it manually, but then for example 30 minutes later I write another script and I want it aliased as well.
So if I could have a script that goes through all scripts in some folder and alias them for me would be very helpful.
I wrote some aliaser using the direct DOM manipulation, but it's a little bit hacky...

All I need is just a possibility to create / delete aliases via script. If it's not possible because of limitations or not wanting to add it. It's fine as well. But maybe then at least add some basic script in documentation to ease it up (though I guess you show some hacking like `eval('document')` to remove the 25GB RAM payload


this is my hack:

```js

/** @param {NS} ns */
export async function main(ns) {
  const get_bin = ns.ls('home');
  const terminalInput = eval('document').getElementById("terminal-input");
  const handler = Object.keys(terminalInput)[1];

  for(const script of get_bin) {
    if(script.startsWith('.bin/')) {
      const pruned = script.replace(".bin/", '').replace(".js", '');
      terminalInput.value = `alias -g ${pruned}="run ${script}"`;
      terminalInput[handler].onChange({ target: terminalInput });
      terminalInput[handler].onKeyDown({ key: 'Enter', preventDefault: () => null });
      ns.tprint(`Aliased: ${script} as ${pruned}`);
    }
  }
}
```

**Comment by d0sboots** [comment]
I think this might be a personal preference thing... I have a number of scripts too, but I prefer to run them directly because I find autocomplete to be far easier to use than aliasing. If I wanted to do an alias but not manually create a bunch of them, I'd probably adopt a "master script" solution.

**Comment by alexzhaohong** [comment]
Using terminalInput seems like a fine work around, since that is already the accepted method for automating `backdoor` in the early game. `alias` seems more like a bash thing than a `ns.alias()` method. I feel like this request could be closed.
